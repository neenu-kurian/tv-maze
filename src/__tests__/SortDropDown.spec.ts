import { render } from '@testing-library/vue';
import SortDropDown from '@/components/SortDropdown.vue';
import { describe, it, expect } from 'vitest';
import { sortOptions } from './testdata';

describe('SortDropDown.vue', () => {
  it('matches snapshot', () => {
    const { container } = render(SortDropDown, {
      props: {
        options: sortOptions,
      },
    });
    expect(container).toMatchSnapshot();
  });
});
