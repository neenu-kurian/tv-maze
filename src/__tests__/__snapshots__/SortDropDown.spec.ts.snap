// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HomeView.vue > matches snapshot 1`] = `
<div>
  <div
    class="relative inline-flex items-center bg-gray-100 rounded-lg px-3 md:min-w-[200px] float-right"
  >
    <select
      class="appearance-none bg-transparent border-none py-2.5 text-sm text-gray-900 w-full cursor-pointer font-medium"
      name="select-sort"
      value="rating-desc"
    >
      
      <option
        value="rating-desc"
      >
        Rating: High to Low
      </option>
      <option
        value="rating-asc"
      >
        Rating: Low to High
      </option>
      
    </select>
    <svg
      aria-hidden="true"
      class="w-4 h-4 text-gray-500 ml-2 shrink-0 pointer-events-none"
      data-slot="icon"
      fill="none"
      stroke="currentColor"
      stroke-width="1.5"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="m19.5 8.25-7.5 7.5-7.5-7.5"
        stroke-linecap="round"
        stroke-linejoin="round"
      />
    </svg>
  </div>
</div>
`;
